### DISCUSSION TITLE
[Bug]: Failed to start add-on

### DISCUSSION THREAD

**Discussion by GittyGrubHub85** [coder/code-server#5356] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS/Web Information

- Web Browser: Chrome
- Local OS: Windows 10
- Remote OS: Ubuntu
- Remote Architecture: amd64:5.1.2
- `code-server --version`: 5.1.2

### Steps to Reproduce

1. 
2. 
3.

### Expected

Based on the docs, it should start after I installed it.

### Actual

It throws a message that it failed to start the add-on based on this image ![](https://i.imgur.com/dQ9wCHr.png)
https://i.imgur.com/dQ9wCHr.png

### Logs

There are no logs unless if I start the add-on.

### Screenshot/Video

![](https://i.imgur.com/9jQqneS.gif)

### Does this issue happen in VS Code or GitHub Codespaces?

- [x] I cannot reproduce this in VS Code.
- [X] I cannot reproduce this in GitHub Codespaces.

### Are you accessing code-server over HTTPS?

- [X] I am using HTTPS.

### Notes

I am using Home Assistant and used its local address to let me run it in the browser. I was trying to follow a [video tutorial](https://youtu.be/sVqyDtEjudk?t=599) from 2020, with the current time of the video when trying to install Studio Code Server. I really don't know how to configure them directly into the add-on because the UI of the video is so different. The configuration looked different and when I edit it, it keeps disappearing and doesn't save upon refresh (which is something that I'll try again if that's the fix). I end up leaving it [empty](https://i.imgur.com/LnVriKK.png) because I don't know anything. When I start the add-on it [fails to start](https://i.imgur.com/dQ9wCHr.png) it's hard to find any good search results on Google and I'm completely stumped to make my own issue here. I hope I can get a good solution for this thanks. 

Updated errors in regards to this issue:
This is what happens when I install Studio Code Server. A message came up to me stated "Install Failed see supervisor logs"
![](https://i.imgur.com/44LUiZm.gif)
from all of those persistent wave of errors, it caused the system to re-boot by itself.

Here's are the logs of the failed installation:
Logger: homeassistant.components.hassio
Source: components/hassio/websocket_api.py:122
Integration: Home Assistant Supervisor ([documentation](https://www.home-assistant.io/integrations/hassio), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+hassio%22))
First occurred: 8:41:52 PM (2 occurrences)
Last logged: 8:42:03 PM

Failed to to call /addons/a0d7b954_vscode/start - Image ghcr.io/hassio-addons/vscode/amd64:5.1.2 does not exist for addon_a0d7b954_vscode

**Comment by jsjoeio** [comment, marked as answer]
To be honest, I have not heard heard of the Studio Code Server thing...is this just a plugin for Home Assistant? Is there some who added it that could help you? I wish I could be more help but I'm not familiar enough.

**Reply by GittyGrubHub85** [reply]
Studio Code Server is Visual Studio Code. I don't know why the devs renamed it but that's what I saw its name when installing it.

I've figured it out myself though. The tutorial was too outdated and it was kind of pointless converting my .vdi to vdi compared to the tutorial it was .vdmk to be converted to .vdi. I decide to find another tutorial that is more updated to the recent HA version (https://www.youtube.com/watch?v=oYjOYkdyr38&ab_channel=GEEKrarGuides)

I deleted that old setup and made a new setup. The newer tutorial did fixed it. This is 2022 note so the 2021 tutorial video worked for me.

**Reply by jsjoeio** [reply]
Hooray! Well I'm glad you figured it out.
